rectools/model_selection/cross_validate.py

```diff
@@ -49,6 +60,7 @@ def cross_validate(  # pylint: disable=too-many-locals
     k: int,
     filter_viewed: bool,
     items_to_recommend: tp.Optional[ExternalIds] = None,
+    keep_features_for_hot_only: bool = False,
```

Comment:
Can we move this to be a `splitter` argument and instance attribute? Not a `cross-validate` parameter? Intuition is that all of the splitting logic must be in one place. Together with all the "filter_cold_" and so on.

Reply:
Apart from `filter_cold_` is there gonna be `filter_warm_` logic in splitter?
In this case `splitter` should probably work with Dataset, not just Interactions.
On the other hand, there can be an additional DatasetSplitter, which can handle the Dataset. And `keep_features_for_hot_only` belongs there.

What's the plan?

Reply:
Discussed separately. Decided to keep it as is for now